data/base-hunting.yaml

```diff
@@ -1,599 +1,604 @@
----
```

Comment:
GitHub seems to think the entire `base-hunting.yaml` file changed. Do you know what happened here?